### PR TITLE
jira-lifecycle-plugin: enable release notes requirement

### PR DESCRIPTION
**Pull request by AlexNPavel**
This PR enables the release notes requirement globally in the jira-lifecycle-plugin.

### CODE REVIEW

File: core-services/jira-lifecycle-plugin/config.yaml [hunk 1]
```diff
@@ -16,6 +16,8 @@ default:
     - resolution: DONE-ERRATA
       status: CLOSED
     is_open: true
+    require_release_notes: true
+    release_notes_default_text: "Cause: What actions or circumstances cause this bug to present.\r\nConsequence: What happens when the bug presents.\r\nFix: What was done to fix the bug.\r\nResult: Bug doesn’t present anymore."
```
Comment (bradmwilliams) [inline core-services/jira-lifecycle-plugin/config.yaml:20]: Do we need to announce this somewhere?

**Review by bradmwilliams — APPROVED** [review]
/hold

File: core-services/jira-lifecycle-plugin/config.yaml (hunk 1 above)
Comment (AlexNPavel) [reply core-services/jira-lifecycle-plugin/config.yaml:20]: The announcement email has been sent and this requirement is ready to be enabled.

**Pull request merged**